code/modules/clothing/head/collectable.dm

```diff
@@ -13,7 +13,8 @@
 /obj/item/clothing/head/collectable/slime
 	name = "collectable slime cap!"
 	desc = "It just latches right in place!"
```

Comment:
imo i think the other desc is funnier (and it is the current des because of compilation order)

Reply:
I think the other desc is more fitting for a headslug instead of a slime, can you get a third person to weigh in?